package/kedro_viz/models/flowchart.py

```diff
@@ -143,15 +143,7 @@ class GraphNode(BaseModel, abc.ABC):
         set(), description="The set of registered pipeline IDs this node belongs to"
     )
 
-    # In Kedro, modular pipeline is implemented by declaring namespace on a node.
-    # For example, node(func, namespace="uk.de") means this node belongs
-    # to the modular pipeline "uk" and "uk.de"
-    namespace: Optional[str] = Field(
-        default=None,
-        validate_default=True,
-        description="The original namespace on this node",
-    )
-    modular_pipelines: Optional[List[str]] = Field(
+    modular_pipeline: Optional[str] = Field(
```

Comment:
So when a GraphNode belongs to a namespace `a.b.c` does that mean that node is only part of `a.b.c` or it is still part of namespace `a`, `a.b` etc ?